static/src/javascripts/projects/common/modules/crosswords/classNames.js

```diff
@@ -0,0 +1,9 @@
+define([], function () {
+
+    return function(props) {
+        return Object.keys(props)
+            .filter(function(f){ return props[f] === true; })
+            .join(' ');
+    };
+
+});
```

Comment:
Elegant 😍 

Since this module has no dependencies, you can drop the empty array from the first argument of the define call.  

```
define(function() { ... })
```